cips/cip-1.md

```diff
@@ -312,11 +312,10 @@ format. There is a [CIP template](./cip-template.md) to follow.
 
 ## CIP Header Preamble
 
-Each CIP must begin with an RFC 822 style header preamble, preceded
-and followed by three hyphens (---). This header is also termed
-“front matter” by Jekyll. The headers must appear in the following order.
-In order to display on the CIP site, the frontmatter must be
-formatted in a markdown table.
+Each CIP must begin with an RFC 822 style header preamble in a
+markdown table. This header is also termed “front matter” by Jekyll.
```

Comment:
[no change needed] we don't use Jekyll so this seems unnecessary

```suggestion
markdown table.
```

but since it existed before this PR, fine to keep.

Reply:
agreed with you, thanks for the review!